src/extraData.ts

```diff
@@ -0,0 +1,54 @@
+import { decode, encode } from '@msgpack/msgpack'
+
+export const MSG_PACK_ID = '0x544c4d50' // hex encoded "TLMP" (Trustlines Message Pack)
+
+export interface ExtraData {
```

Comment:
Why do we put this here and not in `typing.ts`

Reply:
Because it belongs here and not in `typing.ts`. Having a central file is wrong in many ways, most importantly it supports tight coupling of normally separated components. 
Types should be as local as the involving code, to check that these types are  compatible with other parts of the code is the job of the typechecker.

Reply:
I think that extraData handling here was so isolated from the rest, that I think it is fine to break the (wrong) convention of using `typings.ts`